package.json

```diff
@@ -23,8 +23,7 @@
         "import": "./lib/esm/index.js"
     },
     "files": [
-        "dist",
-        "src"
```

Comment:
To deliver the source code to users Git/GitHub should be used. The source files should not exported for a NPM publish.